src/i2s.rs

```diff
@@ -0,0 +1,17 @@
+//! I2S - Inter-IC Sound Interface
+
+use nb;
+
+/// Full duplex
+pub trait FullDuplex<Word> {
+    /// Error type
+    type Error;
+
+    /// Reads the left word and right word available.
+    ///
+    /// The order is in the result is `(left_word, right_word)`
+    fn try_read(&mut self) -> nb::Result<(Word, Word), Self::Error>;
+
+    /// Sends a left word and a right word to the slave.
+    fn try_send(&mut self, left_word: Word, right_word: Word) -> nb::Result<(), Self::Error>;
```

Comment:
Should this be try_write? There's a `read` so a `write` is maybe okay here

Reply:
I named it similarly to `spi::FullDuplex`, where it is also called try_read/[try_send](https://github.com/rust-embedded/embedded-hal/blob/6f172879c7f2a82d97f28f7631b19902f748435d/src/spi.rs#L26)